app/views/users/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<h2>Create a New User</h2>
+
+<%= form_tag "/users", method: "create" do %>
+
+<%= label_tag :first_name %>
+<%= text_field_tag :first_name %>
+
+<%= label_tag :last_name %>
+<%= text_field_tag :last_name %>
+
+<%= label_tag :email %>
+<%= text_field_tag :email %>
+
+<%= label_tag :password %>
+<%= text_field_tag :password %>
+
+<%= label_tag :password_verification %>
+<%= text_field_tag :password_verification %>
```

Comment:
I'm confused where we are confirming that the password and the password verification are matching. I've never used a verification before and I'm not sure how it all works. Can you clarify?